assets/helpers/tracking/__tests__/acquisitionsTest.js

```diff
@@ -37,13 +41,19 @@ describe('acquisitions', () => {
         testId3: 'variant3',
       };
 
+      const optimizeExperiments = {
+        optimizeExperiment1: 'variant1',
+        optimizeExperiment2: 'variant2',
+        optimizeExperiment3: 'variant3',
+      };
+
       const paymentApiAcquisitionData =
-        derivePaymentApiAcquisitionData(referrerAcquisitionData, nativeAbParticipations);
+        derivePaymentApiAcquisitionData(referrerAcquisitionData, nativeAbParticipations, optimizeExperiments);
 
       expect(paymentApiAcquisitionData).toMatchSnapshot();
 
       // The abTests array should be a combination of supportAbTests and the source tests
-      expect(paymentApiAcquisitionData.abTests && paymentApiAcquisitionData.abTests.length).toEqual(4);
+      expect(paymentApiAcquisitionData.abTests && paymentApiAcquisitionData.abTests.length).toEqual(7);
```

Comment:
What does 7 mean? (or 4 for that matter)

Reply:
I believe the point of this test is to assert that the ab tests in the acquisition data is a union of the ab test from the referrering page, the participations and (the newly added) optimize tests. This test could be made clearer and more robust; I'd be happy to do in a subsequent pull request.